libs/ledgerjs/packages/errors/src/index.ts

```diff
@@ -26,6 +26,9 @@ export const CantOpenDevice = createCustomErrorClass("CantOpenDevice");
 export const CashAddrNotSupported = createCustomErrorClass(
   "CashAddrNotSupported"
 );
+export const ClaimRewardsFeesWarning = createCustomErrorClass(
+  "ClaimRewardsFeesWarning"
+);
```

Comment:
Why moving only this one here? No other candidates from `libs/ledger-live-common/src/errors.ts`?
A bit out of scope of this PR anyway.

Reply:
This error has been moved because it is now used by the `coin-algorand` lib and not only `ledger-live-common` as it was the case before.
As part of this migration work, I did not encounter other errors suited for such a move in `ledgerjs/packages/errors`